TwitchClient/Game.swift

```diff
@@ -1,35 +0,0 @@
-//
-//  Game.swift
-//  TwitchClient
-//
-//  Created by Ahmed kotb on 4/7/17.
-//  Copyright © 2017 Ahmed Kotb. All rights reserved.
-//
-
-import UIKit
-
-class Game: NSObject {
-
-    var name : String?
-    var popularity : NSInteger?
-    var boxArt : Image?
-    var logo: Image?
-    var viewers : NSInteger?
-    var channels : NSInteger?
-    
-    func populateGameWithObject(dataDictionary: NSDictionary) -> Game
-    {
-        let gameDictionary : NSDictionary = dataDictionary["game"] as! NSDictionary
```

Comment:
Using `as!` is dangerous. If the JSON returned from network has something wrong then your app would simply crash. It would be better to `guard` or `if let` it.

Topics: force unwrapping in Swift, Swift type safety in general.